src/App.jsx

```diff
@@ -1,12 +1,16 @@
+import { Route, Routes } from 'react-router-dom';
 import { Navbar, Wrapper } from './components/index';
-import { Home } from './pages/Home/Home';
+import { Home, Products } from './pages/index';
```

Comment:
If you are using barrel imports then no need to write **index** it automatically takes.